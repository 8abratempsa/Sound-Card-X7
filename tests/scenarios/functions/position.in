octosql "SELECT position('test', 'es'), position('test', ''), position('test', 'ex'), position('test', 'testt'), position('test', 'test')" --output stream_native
