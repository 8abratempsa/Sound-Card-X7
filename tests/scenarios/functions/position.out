{+0001-01-01T00:00:00Z| 1, 0, <null>, <null>, 0 |}
